### PR TITLE
Adding large_scle_tile bit

### PR DESCRIPTION
**Pull request by tengjinchung**

### CODE REVIEW

File: libavcodec/vaapi_av1.c [hunk 1]
```diff
@@ -241,6 +241,7 @@ static int vaapi_av1_start_frame(AVCodecContext *avctx,
             .disable_frame_end_update_cdf = frame_header->disable_frame_end_update_cdf,
             .uniform_tile_spacing_flag    = frame_header->uniform_tile_spacing_flag,
             .allow_warped_motion          = frame_header->allow_warped_motion,
+            .large_scale_tile             = frame_header->large_scale_tile,
```
Comment (xhaihao) [inline libavcodec/vaapi_av1.c:244]: Please add a check for va-api version at compile time. large_scale_tile is not available in old versions of libva.

Comment (xhaihao) [reply libavcodec/vaapi_av1.c:244]: Please ignore the above comment.

**Review by xhaihao — APPROVED** [review]
LGTM.

**Comment by xhaihao** [comment]
> LGTM.

Seems large_scale_tile is not handled in cbs.

**Review by xhaihao — CHANGES_REQUESTED** [review]
need read / write large_scale_tile in cbs.

**Comment by xhaihao** [comment]
@tengjinchung  May I close this PR ?

**Comment by tengjinchung** [comment]
> @tengjinchung May I close this PR ?

sure.

**Pull request closed without merging**